packages/pytest-simcore/src/pytest_simcore/playwright_utils.py

```diff
@@ -102,3 +197,27 @@ def wait_for_pipeline_state(
             )
         print(f"<-- pipeline is in {current_state=}")
     return current_state
+
+
+def on_web_socket(ws) -> None:
```

Comment:
Q: Does this function belong to the common utils? This was just a first try where we log everything that comes through the websocket, but I guess it might be different on the test-to-test use case.

Reply:
It is used in several playwrite tests so I moved it here
For the moment there is no other function dealing with sockets